docs/content/preview/benchmark/ycsb-ysql.md

```diff
@@ -94,108 +84,102 @@ db.user=yugabyte
 db.passwd=
 ```
 
-Details about other configuration parameters are described in [Core Properties](https://github.com/brianfrankcooper/YCSB/wiki/Core-Properties) on the [YCSB](https://github.com/brianfrankcooper/YCSB/) repository.
+The other configuration parameters are described in [Core Properties](https://github.com/brianfrankcooper/YCSB/wiki/Core-Properties).
 
-{{< note title="Note" >}}
-The db.url field should be populated with the IPs of all the nodes that are part of the cluster.
-{{< /note >}}
+### Run the benchmark
 
-### 4. Run the benchmark
-
-There is a handy script `run_ysql.sh` that loads and runs all the workloads.
+Use the following script `run_ysql.sh` to load and run all the workloads:
 
 ```sh
 $ ./run_ysql.sh --ip <ip>
 ```
 
-The above command workload will run the workload on a table with 1 million rows. If you want to run the benchmark on a table with a different row count:
+The preceding command runs the workload on a table with a million rows. To run the benchmark on a table with a different row count, use the following command:
 
 ```sh
 $ ./run_ysql.sh --ip <ip> --recordcount <number of rows>
 ```
 
-{{< note title="Note" >}}
-To get the maximum performance out of the system, you would have to tune the threadcount parameter in the script. As a reference, for a c5.4xlarge instance with 16 cores and 32GB RAM, you used a threadcount of 32 for the loading phase and 256 for the execution phase.
-{{< /note >}}
+To get the maximum performance out of the system, you would have to tune the `threadcount` parameter in the script. As a reference, for a c5.4xlarge instance with 16 cores and 32GB RAM, you used a `thread count` of 32 for the loading phase and 256 for the execution phase.
 
-### 5. Verify results
+### Verify results
 
-The script creates 2 result files per workload, one for the loading and one for the execution phase with the details of throughput and latency.
-For example for `workloada` it creates `workloada-ysql-load.dat` and `workloada-ysql-transaction.dat`
+The `run_ysql.sh` script creates two result files per workload: one for the loading, and one for the execution phase with the details of throughput and latency.
 
-### 6. Run individual workloads (optional)
+For example, for a workload it creates, inspect the `workloada-ysql-load.dat` and `workloada-ysql-transaction.dat` files.
 
-Connect to the database using `ysqlsh`.
+### Run individual workloads (optional)
 
-```sh
-$ ./bin/ysqlsh -h <ip>
-```
+Optionally, you can run workloads individually using the following steps:
 
-Create the `ycsb` database.
+1. Start the YSQL shell using the following command:
 
-```sql
-yugabyte=# CREATE DATABASE ycsb;
-```
+    ```sh
+    $ ./bin/ysqlsh -h <ip>
+    ```
 
-Connect to the created database.
+1. Create the `ycsb` database as follows:
 
-```sql
-yugabyte=# \c ycsb
-```
+    ```sql
+    yugabyte=# CREATE DATABASE ycsb;
+    ```
 
-Create the table.
+1. Connect to the database** as follows:
```

Comment:
what are the asterisks for

Reply:
Connect to the database was in bold earlier , forgot to remove this. Thanks. Will fix it.